options/options.html

```diff
@@ -6,13 +6,28 @@
 
 <body>
   <label>
-    <input type="number" placeholder="32" id="MB">
-    Maximum available decoding memory in Megabytes (MB)
+    <select id="DecoderMemoryLimitInMegabytes">
+      <option value=2>2 MB</option>
+      <option value=4>4 MB</option>
+      <option value=8>8 MB</option>
+      <option value=16>16 MB</option>
+      <option value=32>32 MB</option>
+      <option value=64>64 MB</option>
+      <option value=128>128 MB</option>
+      <option value=256>256 MB</option>
+    </select>
+    Maximum available decoding memory in Megabytes
+  </label>
+  <label>
+    <BR>
+    <!-- Must restart due to loading of background.js only once -->
+    <input type="checkbox" id="DebugOutput" title="Browser must be restarted for this to take effect.">
```

Comment:
Strange ... If the value is persisted in local storage and read back when the extension starts it should work. Are you sure you are reading it in when background.js is processed the first time? I guess that is when you'd have to set it ...